src/graphnet/utils/cluster/condor.py

```diff
@@ -0,0 +1,843 @@
+"""
+Tools for submitting to a cluster via the condor system.
+Also supports DAGMan.
+
+These tools are generic to any condor system, e.g. do not assume IceCube, NPX, etc.
+
+These can be used standalone, or by the more sophisticated job management system 
+defined in `cluster.py` and `job.py`.
+
+Tom Stuttard
+"""
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import unicode_literals
+
+from builtins import open
+from builtins import int
+from builtins import str
+from builtins import object
+from future import standard_library
+
+standard_library.install_aliases()
+import os
+
+from graphnet.utils.cluster.filesys_tools import make_dir, is_executable
+from graphnet.utils.cluster.unix_tools import tail
+
+from graphnet.utils.cluster.job import JOB_INDEX_FMT, JOB_NAME_FMT
+
+#
+# Globals
+#
+
+CONDOR_SUBMIT_EXE = "condor_submit_dag"
+
+#
+# Submission
+#
+
+
+def percentage(numerator, denominator):
+    return 100.0 * fraction(numerator, denominator)
+
+
```

Comment:
I don't see this function referenced elsewhere (apart from one or more places that are not themselves references elsewhere)?
```suggestion
```